diff --git a/frontend/babel.config.js b/frontend/babel.config.js
index 2900afe..db538eb 100644
--- a/frontend/babel.config.js
+++ b/frontend/babel.config.js
@@ -1,6 +1,11 @@
 module.exports = function(api) {
   api.cache(true);
   return {
     presets: ['babel-preset-expo'],
+    env: {
+      production: {
+        plugins: [['transform-remove-console', { exclude: ['error', 'warn'] }]]
+      }
+    }
   };
 };